### DISCUSSION TITLE
How to stream the screen in RTMP URL using RPBroadcastSampleHandler?

### DISCUSSION THREAD

**Discussion by rishopbabu** [HaishinKit/HaishinKit.swift#1142] [Q&A]
@shogo4405 

- For **RPBroadcastSampleHandler**  I have tried the below steps. 

**Step1**:   Created **Broadcast upload extension** in the name of (screenShare ) in Xcode 
**Step2**:   it creates a folder(screenShare) with the file name **SampleHandler** 

- In SampleHandler there are **broadcastStarted, broadcastPaused, and processSampleBuffer**, etc.
- Even though I used this  **RPSystemBroadcastPickerView** to start the broadcast not trigger any methods.
- **So When these methods will trigger how can I invoke these for the screen share with audio using the Broadcast upload extension?**

**Note:** 
            I can see there is a broadcast upload extension(Screencast) in the Haishinkit iOS project. It will be very helpful to guide How to use that.

<img width="1122" alt="MicrosoftTeams-image" src="https://user-images.githubusercontent.com/102947456/220897075-c3d0d1f0-269b-4a96-b28c-8f2169708370.png">

**Comment by shogo4405** [comment]
1.1  Please open the [Setting.app]
1.2. Move to the [Control Center].
1.3. Add the [Screen Recording] 
2.1. Back to home screen.
2.2. Pull down Control Center.
2.3. You can see Screen Recorsing feature with icon like ◎. Please long tap.
2.4. Select "HaishinKit" and start to broadcast.
